NuGet.Clients.sln

```diff
@@ -72,185 +72,752 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "UnitTests", "UnitTests", "{
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "NuGet.CommandLine.Test", "test\NuGet.Clients.Tests\NuGet.CommandLine.Test\NuGet.CommandLine.Test.csproj", "{33294BB8-5144-4363-B02C-332F6D94C1C1}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "PackageManagerProvider.Test", "test\NuGet.Clients.Tests\NuGet.PackageManagement.UI.Test\PackageManagerProvider.Test.csproj", "{1E243B39-7C61-4525-A664-3DC8589707AC}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Coverage|Any CPU = Coverage|Any CPU
+		Coverage|ARM = Coverage|ARM
+		Coverage|x64 = Coverage|x64
+		Coverage|x86 = Coverage|x86
 		Debug|Any CPU = Debug|Any CPU
+		Debug|ARM = Debug|ARM
+		Debug|x64 = Debug|x64
+		Debug|x86 = Debug|x86
 		Mono Debug|Any CPU = Mono Debug|Any CPU
+		Mono Debug|ARM = Mono Debug|ARM
+		Mono Debug|x64 = Mono Debug|x64
+		Mono Debug|x86 = Mono Debug|x86
 		Mono Release|Any CPU = Mono Release|Any CPU
+		Mono Release|ARM = Mono Release|ARM
+		Mono Release|x64 = Mono Release|x64
+		Mono Release|x86 = Mono Release|x86
 		Release|Any CPU = Release|Any CPU
+		Release|ARM = Release|ARM
+		Release|x64 = Release|x64
+		Release|x86 = Release|x86
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
 		{306CDDFA-FF0B-4299-930C-9EC6C9308160}.Coverage|Any CPU.ActiveCfg = Release|Any CPU
 		{306CDDFA-FF0B-4299-930C-9EC6C9308160}.Coverage|Any CPU.Build.0 = Release|Any CPU
+		{306CDDFA-FF0B-4299-930C-9EC6C9308160}.Coverage|ARM.ActiveCfg = Debug|Any CPU
```

Comment:
What happened here? This doesn't look wrong, but why were these added as part of this?